assets/src/components/detours/diversionPage.tsx

```diff
@@ -20,6 +23,7 @@ export const DiversionPage = ({
   originalRoute,
   onClose,
 }: DiversionPageProps) => {
+  const [detourFinished, setDetourFinished] = useState(false)
```

Comment:
I'm going to follow this PR up with another one which integrates this state into `useDetour`